docs/reference/crds/policies.md

```diff
@@ -5,22 +5,24 @@ View more CRD samples [here](../../../config/samples/policy). The `metadata.name
 apiVersion: config.ratify.deislabs.io/v1alpha1
 kind: Policy
 metadata:
-  name: "configpolicy"
```

Comment:
nit: curious if adding a `-` might be more readable? so `config-policy`?

Reply:
good call, now we support config-policy, configpolicy and configPolicy.

Reply:
great!